numpy/core/fromnumeric.py

```diff
@@ -2282,11 +2298,28 @@ def amax(a, axis=None, out=None, keepdims=np._NoValue):
     >>> np.nanmax(b)
     4.0
 
+    You can use an initializer to compute the maximum of an empty slice, or
+    to initialize it to a different value:
+
+    >>> np.max([], initializer=-np.inf)
+    -inf
+    >>> np.max([50], initializer=25)
+    50
+
+    Notice that this isn't the same as Python's ``default`` argument.
```

Comment:
Be explicit in what is different: "Notice that the initializer is used as one of the elements for which the maximum is determined, unlike for the ``default`` argument Python's `max` function, which is only used for empty iterables."